### PR TITLE
Fraction

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
Validating field if before "."/"," has 10 digits, and after it 5 digits.
valid:
fractionvalidation
invalid:
fractionvalidation2

### CODE REVIEW

File: fraction/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Fraction</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
```
Comment (panayotkulchev) [inline fraction/index.html]: Will everything be OK if bootrap and font awsome CDNs are down for some reason? I mean you can inport external libraries in the project and exclude in .gitignore . This is only for your information. If you wan you can try it. 
This way you exclude two dependencies from the project.

**Comment by panayotkulchev** [comment]
Please for each topic create both `reviewed` and `needs review` labels. And follow the same colors - green for needs review and blue for reviewed.

File: fraction/index.html (hunk 1 above)
Comment (KrasimirRaikov) [inline fraction/index.html]: Done.

**Comment by panayotkulchev** [comment]
LGTM!

**Pull request merged**